dwave_structural_imbalance_demo/interfaces.py

```diff
@@ -18,6 +21,18 @@ def iteritems(d): return d.iteritems()
     def iteritems(d): return d.items()
 
 
+@lru_cache(maxsize=None)
```

Comment:
Because this is only ever used in the tests, IMO it should probably just live in `tests/__init__.py`.

Reply:
Agreed, and done!